doc/2/guides/introduction/2-general-purpose-backend/index.md

```diff
@@ -8,15 +8,103 @@ order: 200
 
 # General Purpose Backend
 
-Kuzzle is a generic backend offering features common to all applications.
+Kuzzle is a generic backend offering **features common to every application**.
 
-Rather than redeveloping functionalities that don't bring any added value, Kuzzle proposes to use them as is to build its high-level business functionalities.
+Rather than redeveloping functionalities that don't bring any added value, Kuzzle proposes to use them as is to **build high-level business functionalities**.
```

Comment:
I find this sentence really weird...

Reply:
wdyt if I add `basics`
> Rather than redeveloping `basics` functionalities that don't bring any added value, Kuzzle proposes to use them as is to **build high-level business functionalities**.

Reply:
Hm... And what about this proposal?
(ping @xbill82 )

> Rather than developing the same standard features over and over again each time you create a new application, Kuzzle proposes them off-the-shelf, ready to be used, and allowing you to focus on **building high-level, high-value business functionalities**.

Reply:
I really like this one